Makefile

```diff
@@ -94,6 +94,11 @@ check-workflows: build ## validate Workflows
 		bash bin/validate-json.sh
 .PHONY: check-workflows
 
+check-workflow-tags: build ## validate Workflow tags
+	$(ACTIVATE_ENV) && \
+		bash bin/validate-workflow-tags.sh
```

Comment:
if you move your check to the `check-workflows` target above it, it will be included in the travis linting

Reply:
@shiltemann like:

```Makefile
check-workflows: build ## validate Workflows
	$(ACTIVATE_ENV) && \
		bash bin/validate-json.sh
		bash bin/validate-workflow-tags.sh
.PHONY: check-workflows
```

or
```Makefile
check-workflows: build ## validate Workflows
	$(ACTIVATE_ENV) && \
		bash bin/validate-json.sh
	$(ACTIVATE_ENV) && \
		bash bin/validate-workflow-tags.sh
.PHONY: check-workflows
```

(is my first time using a Makefile)